### PR TITLE
Enable parallel npm script execution with npm-run-all

### PR DESCRIPTION
**Pull request by jprinet**
Without parallel execution
```shell
> time npm run all
npm run all  35.53s user 2.79s system 280% cpu 13.679 total
```

with concurrently
```shell
> time npm run all
npm run all  45.48s user 3.44s system 402% cpu 12.158 total
```

with npm-run-all --parallel
```shell
> time npm run all
npm run all  41.31s user 3.19s system 498% cpu 8.922 total
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,11 +25,11 @@
     "package-maven-build-scan-capture-extension": "mvn clean package -f maven-build-scan-capture-extension/pom.xml",
     "package-build-scan-setup-maven-main": "ncc build build-scan-setup-maven/src/main.ts --out build-scan-setup-maven/dist/main",
     "package-build-scan-setup-maven-post": "ncc build build-scan-setup-maven/src/post.ts --out build-scan-setup-maven/dist/post",
-    "package-build-scan-setup-maven": "npm run package-build-scan-setup-maven-main && npm run package-build-scan-setup-maven-post && npm run package-maven-build-scan-capture-extension",
+    "package-build-scan-setup-maven": "npm-run-all --parallel package-build-scan-setup-maven-main package-build-scan-setup-maven-post package-maven-build-scan-capture-extension",
     "package-build-scan-setup": "npm run package-build-scan-setup-maven",
```
Comment (bigdaz) [inline package.json:29]: I'm not sure why you have extra steps of redirection like this. I'd remove this until it proves necessary/useful.

File: package.json [hunk 2]
```diff
@@ -25,11 +25,11 @@
...
     "package-build-scan-publish-maven": "ncc build build-scan-publish-maven/src/main.ts --out build-scan-publish-maven/dist",
     "package-build-scan-publish": "npm run package-build-scan-publish-maven",
-    "package-build-scan": "npm run package-build-scan-setup && npm run package-build-scan-publish",
+    "package-build-scan": "npm-run-all --parallel package-build-scan-setup package-build-scan-publish",
```
Comment (bigdaz) [inline package.json:32]: The `npm-run-all` tool accepts wildcard arguments, so you can probably reorganize this to be simpler.
Something like:
```
"package": "npm-run-all --parallel package-*"
```

I also find it useful to have a `npm run build` command that runs `format:write`, `lint` and `package`, but doesn't run any tests.

File: package.json [hunk 3]
```diff
@@ -25,11 +25,11 @@
...
     "test": "jest",
     "all": "npm run format:write && npm run lint && npm run package-build-scan && npm run test"
```
Comment (bigdaz) [inline package.json:34]: I'm running `format:write` and `lint` in parallel as well, with a "goal" called `check`.

**Review by bigdaz — COMMENTED** [review]
Looks fine to me. I left some suggestions for possible improvements.

File: package.json (hunk 1 above)
Comment (jprinet) [reply package.json:29]: This is a left over from the Gradle/Maven cohabitation. It is not [yet] required indeed and I removed it 👍

File: package.json (hunk 2 above)
Comment (jprinet) [reply package.json]: I applied your suggestion 👍

**Pull request merged**